src/components/Theme/style.css

```diff
@@ -121,7 +121,7 @@
 .help {
   padding: 1.5rem 1rem 1rem;;
   text-align: left;
-  box-shadow: inset 0 0 0px 1px #8A9293;
+  box-shadow: inset 0 0 0px 1px rgba(138,146,147,0.35);
```

Comment:
fwiw, you should be able to do `rgba(#8A9293, 0.35)` rather than mixing rgb and hex

Reply:
It breaks if I try to do `rgba(#8A9293, 0.35)`

Reply:
Ahh, I assumed that this was SCSS - but it's actually just vanilla CSS.

Yeah what I said is a SCSS thing

Reply:
it's less